examples/hotelling_law/cprofile_test.py

```diff
@@ -0,0 +1,34 @@
+import cProfile
```

Comment:
Forgot to comment this: the file cprofile_test.py shouldn't be part of the example.

Reply:
ok. removing this file

Reply:
remvoed